wallet/db_postgres.c

```diff
@@ -1,5 +1,6 @@
 #include "db_postgres_sqlgen.c"
 #include <ccan/ccan/tal/str/str.h>
+#include <ccan/compiler/compiler.h>
```

Comment:
unable to understand the motivation that we need the `ccan/compiler` here

Reply:
`UNUSED` macro.